### DISCUSSION TITLE
Help getting this dalle-playground running on Colab

### DISCUSSION THREAD

**Discussion by GreekPhysique** [saharmor/dalle-playground#96] [General]
I tried the code and it never gave me a url to drop int to the front end

Could someone please help me? i'm not sure how to get the URL?

Thank you in advanced

![Screenshot 2022-06-24 092412](https://user-images.githubusercontent.com/91274503/175545118-928af036-c14e-49cf-bd66-1b5d862bfae1.png)

**Comment by Aradus** [comment]
same :/
